conf/build.manifest

```diff
@@ -189,6 +189,12 @@ additional-packages:
   comment: NAS-114723
 - package: powertop
   comment: requested by community (NAS-113898)
+- package: rdma-core
+  comment: requested by community (NAS-115260)
+- package: ibutils
+  comment: requested by community (NAS-115260)
+- package: infiniband-diags
+  comment: requested by community (NAS-115260)
```

Comment:
Just to make sure, have we verified we are free to distribute those? e.g. not non-free

Reply:
Yes, they are non-free. I verified that via debian tracker